test/unit/search/fm_index_cursor/extend_bitpacked_query_sequence_test.cpp

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2006-2024 Knut Reinert & Freie Universität Berlin
+// SPDX-FileCopyrightText: 2016-2024 Knut Reinert & MPI für molekulare Genetik
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
```

Comment:
```suggestion
```

Needed?

Reply:
Well, I am using a `std::vector`. It might be included by the `helper.hpp` already. Not including it has the potential that if some other code is refactored this test might not compile anymore since vector is not included through some dependent includes. Adding it makes clear, that I need it in the test.
But frankly, I don't care too much. What is the preferred policy here?

Reply:
Ah, didn't see the `result_t`. It's fine then